js/audio/player/AudioPlayer.svelte

```diff
@@ -258,12 +370,16 @@
 			bind:mode
 			bind:trimDuration
 			bind:show_volume_slider
+			bind:subtitles_toggle
 			show_redo={interactive}
 			{handle_reset_value}
 			{waveform_options}
 			{trim_region_settings}
 			{editable}
+			show_subtitles={subtitles !== null}
 		/>
+		<audio bind:this={audio_player} preload="metadata" style="display: none;"
```

Comment:
Why do we need this second audio player here? We already have the `standard-player` above